docs/0.4.x/en-US/capsules/intro.md

```diff
@@ -8,12 +8,12 @@ The reason capsules are useful is because they can be cached very aggressively:
 
 ## Capsules vs. templates
 
-Templates create page, and capsules create widgets, but the major differences between them are how they handle errors, how they're displayed, and how they're requested. On an *initial load* (wehn a user comes to your site from somewhere else on the internet), Perseus will send your page, together with all its capsules, down to the browser as one great big HTML file, containing all the states necessary. When Perseus starts up, it will aggressively cache those widgets and link them to the page, so that, if that page is ever dropped from the internal cache, those widgets will be too (provided they're not being used by any other pages). When a user goes to another page in your app (using the *subsequent load* system), Perseus will send down only the state of the new page, and the client will then request all the widgets. This can lead to much faster page loads: most of the content of the page is sent down straight away, but the widgets are loaded separately (but all in parallel). While they're loading, you can specify a fallback state for each one, which might be a *skeleton* (those pulsating bars that indicate "there will be text here soon"), or a simple loading icon, or anything else you can dream up.
+Templates create page, and capsules create widgets, but the major differences between them are how they handle errors, how they're displayed, and how they're requested. On an *initial load* (when a user comes to your site from somewhere else on the internet), Perseus will send your page, together with all its capsules, down to the browser as one great big HTML file, containing all the states necessary. When Perseus starts up, it will aggressively cache those widgets and link them to the page, so that, if that page is ever dropped from the internal cache, those widgets will be too (provided they're not being used by any other pages). When a user goes to another page in your app (using the *subsequent load* system), Perseus will send down only the state of the new page, and the client will then request all the widgets. This can lead to much faster page loads: most of the content of the page is sent down straight away, but the widgets are loaded separately (but all in parallel). While they're loading, you can specify a fallback state for each one, which might be a *skeleton* (those pulsating bars that indicate "there will be text here soon"), or a simple loading icon, or anything else you can dream up.
```

Comment:
```suggestion
Templates create pages, and capsules create widgets, but the major differences between them are how they handle errors, how they're displayed, and how they're requested. On an *initial load* (when a user comes to your site from somewhere else on the internet), Perseus will send your page, together with all its capsules, down to the browser as one great big HTML file, containing all the states necessary. When Perseus starts up, it will aggressively cache those widgets and link them to the page, so that, if that page is ever dropped from the internal cache, those widgets will be too (provided they're not being used by any other pages). When a user goes to another page in your app (using the *subsequent load* system), Perseus will send down only the state of the new page, and the client will then request all the widgets. This can lead to much faster page loads: most of the content of the page is sent down straight away, but the widgets are loaded separately (but all in parallel). While they're loading, you can specify a fallback state for each one, which might be a *skeleton* (those pulsating bars that indicate "there will be text here soon"), or a simple loading icon, or anything else you can dream up.
```

Reply:
Thank you!